apps/angular/src/app/features/home/anime-table/anime-table.component.html

```diff
@@ -0,0 +1,61 @@
+@if ({
+	results: (animeList$ | async)?.results || []
+}; as animeList) {
+	<table mat-table [dataSource]="animeList.results" class="table" [trackBy]="trackAnimeById">
+		<ng-container [matColumnDef]="animeColumns.Index">
+			<th mat-header-cell *matHeaderCellDef>#</th>
+			<td mat-cell *matCellDef="let anime; let i = index">{{ i + 1 }}</td>
+		</ng-container>
+
+		<ng-container [matColumnDef]="animeColumns.Image">
+			<th mat-header-cell *matHeaderCellDef>Image</th>
+			<td mat-cell *matCellDef="let anime">
+				<div class="table__row-image-container">
+					<img class="table__row-image" [src]="anime.imageUrl" [alt]="animeImageDescription(anime)">
+				</div>
+			</td>
+		</ng-container>
+
+		<ng-container [matColumnDef]="animeColumns.EnglishTitle">
+			<th class="table__column_wide" mat-header-cell *matHeaderCellDef>English Title</th>
+			<td class="table__column_wide" mat-cell *matCellDef="let anime">{{ anime.englishTitle | appNullable }}</td>
+		</ng-container>
+
+		<ng-container [matColumnDef]="animeColumns.JapaneseTitle">
+			<th class="table__column_wide" mat-header-cell *matHeaderCellDef>Japanese Title</th>
+			<td class="table__column_wide" mat-cell *matCellDef="let anime">{{ anime.japaneseTitle | appNullable }}</td>
+		</ng-container>
+
+		<ng-container [matColumnDef]="animeColumns.AiredStart">
+			<th mat-header-cell *matHeaderCellDef>Start Date</th>
+			<td mat-cell *matCellDef="let anime">{{ anime.aired.startAt | date:dateFormat | appNullable }}</td>
+		</ng-container>
+
+		<ng-container [matColumnDef]="animeColumns.Type">
+			<th mat-header-cell *matHeaderCellDef>Type</th>
+			<td mat-cell *matCellDef="let anime">{{ anime.type | appNullable }}</td>
+		</ng-container>
+
+		<ng-container [matColumnDef]="animeColumns.Status">
+			<th mat-header-cell *matHeaderCellDef>Status</th>
+			<td mat-cell *matCellDef="let anime">{{ anime.status | appNullable }}</td>
+		</ng-container>
+
+		<tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+		<tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
+
+		<tr class="mat-row" *matNoDataRow>
+			<td class="mat-cell table__no-data-row" colspan="7">
+				@if (isLoading$ | async) {
+					<camp-progress-spinner />
+				}
+				@if (error$ | async; as errorMessage) {
+					<camp-error-message [errorMessage]="errorMessage" />
+				}
+				@if (isNoData(isLoading$ | async, error$ | async, animeList.results)) {
+					<p slot="table-skeleton-content">There is no anime to display.</p>
+				}
```

Comment:
Usually, we're striving to have only one subscription per stream. To achieve this, we use data objects.

```suggestion
				@if ({
					errorMessage: error$ | async,
					isLoading: isLoading$ | async,
				}; as data) {
					@if (data.isLoading) {
						<camp-progress-spinner />
					}
					@if (data.errorMessage) {
						<camp-error-message [errorMessage]="errorMessage" />
					}
					@if (isNoData(data.isLoading, data.errorMessage, animeList.results)) {
						<p slot="table-skeleton-content">There is no anime to display.</p>
					}
				}
```

FYI, in newer versions (18.1+) angular introduced `@let` construction, that greatly simplifies it and reduces indentations.